llvm/test/CodeGen/DirectX/CreateHandle.ll

```diff
@@ -14,7 +14,7 @@ target triple = "dxil-pc-shadermodel6.0-compute"
 
 declare i32 @some_val();
 
-define void @test_buffers() {
+define void @test_buffers() #0 {
```

Comment:
If we allow zero entry points in tests we don't need this.

Reply:
> If we allow zero entry points in tests we don't need this.

Reverted.